### PR TITLE
migration: Add listen address case

### PR DESCRIPTION
**Pull request by cliping**
VIRT-297874 - VM live migration with copy storage - network data transport - TCP - --listen-address

### CODE REVIEW

**Comment by cliping** [comment]
Test result:

 (01/20) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tcp_listen_address.copy_storage_all.target_ipv4_address.p2p: PASS (222.59 s)
 (02/20) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tcp_listen_address.copy_storage_all.target_ipv4_address.non_p2p: PASS (225.27 s)
 (03/20) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tcp_listen_address.copy_storage_all.target_ipv6_address.p2p: PASS (225.69 s)
 (04/20) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tcp_listen_address.copy_storage_all.target_ipv6_address.non_p2p: PASS (225.75 s)
 (05/20) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tcp_listen_address.copy_storage_all.all_ipv4.p2p: PASS (225.84 s)
 (06/20) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tcp_listen_address.copy_storage_all.all_ipv4.non_p2p: PASS (225.18 s)
 (07/20) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tcp_listen_address.copy_storage_all.all_ipv6.p2p: PASS (225.23 s)
 (08/20) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tcp_listen_address.copy_storage_all.all_ipv6.non_p2p: PASS (223.43 s)
 (09/20) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tcp_listen_address.copy_storage_all.invalid_address.p2p: PASS (203.08 s)
 (10/20) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tcp_listen_address.copy_storage_all.invalid_address.non_p2p: PASS (201.99 s)
 (11/20) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tcp_listen_address.copy_storage_inc.target_ipv4_address.p2p: PASS (225.87 s)
 (12/20) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tcp_listen_address.copy_storage_inc.target_ipv4_address.non_p2p: PASS (225.30 s)
 (13/20) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tcp_listen_address.copy_storage_inc.target_ipv6_address.p2p: PASS (225.69 s)
 (14/20) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tcp_listen_address.copy_storage_inc.target_ipv6_address.non_p2p: PASS (225.99 s)
 (15/20) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tcp_listen_address.copy_storage_inc.all_ipv4.p2p: PASS (226.13 s)
 (16/20) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tcp_listen_address.copy_storage_inc.all_ipv4.non_p2p: PASS (225.89 s)
 (17/20) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tcp_listen_address.copy_storage_inc.all_ipv6.p2p: PASS (224.74 s)
 (18/20) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tcp_listen_address.copy_storage_inc.all_ipv6.non_p2p: PASS (223.21 s)
 (19/20) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tcp_listen_address.copy_storage_inc.invalid_address.p2p: PASS (201.91 s)
 (20/20) type_specific.io-github-autotest-libvirt.migration_with_copy_storage.network_data_transport.tcp_listen_address.copy_storage_inc.invalid_address.non_p2p: PASS (202.84 s)

File: provider/migration/migration_base.py [hunk 1]
```diff
@@ -726,3 +726,14 @@ def get_vm_serial_session_on_dest(params):
     vm_session = migration_obj.vm.wait_for_serial_login(timeout=120)
     params.update({"vm_session": vm_session})
     migration_obj.vm.connect_uri = backup_uri
+
+
+def check_listen_address(params):
```
Comment (Yingshun) [inline provider/migration/migration_base.py:731]: Can it be used by other tests? how about moving to avocado-vt?

**Comment by cliping** [comment]
Depend on: https://github.com/avocado-framework/avocado-vt/pull/3809

File: provider/migration/migration_base.py (hunk 1 above)
Comment (cliping) [reply provider/migration/migration_base.py]: Updated, thanks.

**Review by Yingshun — APPROVED** [review]

**Review by chloerh — APPROVED** [review]

**Pull request merged**